website/content/docs/configuration/session-recording/configure-storage-policy.mdx

```diff
@@ -0,0 +1,660 @@
+   Scope information:
+     Created Time:        Thu, 25 Jan 2024 21:22:45 MST
+     ID:                  o_aDkVBCDTvY
+     Name:                prod-databases
+     Storage Policy ID:   pst_WZ3SQSSYJY
+     Updated Time:        Thu, 25 Jan 2024 22:00:27 MST
+     Version:             7
+   
+     Scope (parent):
+       ID:                global
+       Name:              global
+       Type:              global
+   
+     Authorized Actions:
+       detach-storage-policy
+       no-op
+       read
+       update
+       delete
+       attach-storage-policy
+    ...
+    ... More Output ...
+    ...
+   ```
+
+   </CodeBlockConfig>
+
+   Verify that the `soc2-policy` ID (`pst_WZ3SQSSYJY` in this example) is listed under **Storage Policy ID**.
+
+</Tab>
+<Tab heading="API call using cURL" group="api">
+
+The following API call is an example of reading the details of the `prod-databases` scope by ID (`o_aDkVBCDTvY` in this example).
+
+First, obtain the storage policy ID.
+
+```shell-session
+$ curl --header "Content-Type: application/json" \
+    --header "Authorization: Bearer $(boundary config get-token)" \
+    --request GET \
+    $BOUNDARY_ADDR/v1/scopes/o_aDkVBCDTvY | jq
+```
+
+**Example output:**
+
+<CodeBlockConfig hideClipboard highlight="1-9,61">
+
+```plaintext
+{
+  "id": "o_aDkVBCDTvY",
+  "scope_id": "global",
+  "scope": {
+    "id": "global",
+    "type": "global",
+    "name": "global",
+    "description": "Global Scope"
+  },
+  "name": "prod-databases",
+  "created_time": "2024-01-26T04:22:45.891800Z",
+  "updated_time": "2024-01-26T05:00:27.333892Z",
+  "version": 7,
+  "type": "org",
+  "authorized_actions": [
+    "update",
+    "delete",
+    "attach-storage-policy",
+    "detach-storage-policy",
+    "no-op",
+    "read"
+  ],
+  "authorized_collection_actions": {
+    "auth-methods": [
+      "create",
+      "list"
+    ],
+    "auth-tokens": [
+      "list"
+    ],
+    "groups": [
+      "create",
+      "list"
+    ],
+    "policies": [
+      "list",
+      "create"
+    ],
+    "roles": [
+      "list",
+      "create"
+    ],
+    "scopes": [
+      "list",
+      "list-keys",
+      "rotate-keys",
+      "list-key-version-destruction-jobs",
+      "destroy-key-version",
+      "create"
+    ],
+    "session-recordings": [
+      "list"
+    ],
+    "storage-buckets": [
+      "create",
+      "list"
+    ],
+    "users": [
+      "create",
+      "list"
+    ]
+  },
+  "storage_policy_id": "pst_WZ3SQSSYJY"
+}
+```
+
+</CodeBlockConfig>
+
+Verify that the `soc2-policy` ID (`pst_WZ3SQSSYJY` in this example) is listed under `storage_policy_id`.
+
+</Tab>
+</Tabs>
+
+### Read and list session recordings
+
+New session recordings under the `prod-databases` scope should now show a `retain_until` and `delete_after` date corresponding to the `soc2-policy` storage policy.
+
+<Tabs>
+<Tab heading="UI" group="ui">
+
+1. Create a new session recording on a target within the `prod-databases` org.
```

Comment:
Is this first step right? I think the CLI and API procedures just start with a `list`, which I believe would be the same as clicking Session Recordings in the nav panel.

Reply:
hmm no this is right, I should probably add that step to the other tabs 👍

Reply:
this should be fixed, great catch!